frontend/src/lobby/Lobby.tsx

```diff
@@ -25,6 +25,7 @@ export const isHost = (props: {
 };
 
 const Lobby: Component<{
+  players: Accessor<TPlayer | null>
```

Comment:
```suggestion
  players: Accessor<TPlayer[] | null>
```
i think this should  be a player  list or the variable is named wrongly